template/web/package.json

```diff
@@ -15,11 +15,12 @@
     "@sanity/block-content-to-react": "^2.0.7",
     "@sanity/image-url": "^0.140.18",
     "date-fns": "^1.30.1",
-    "gatsby": "^3.0.4",
+    "gatsby": "^4.4.0-next.0",
```

Comment:
we probably want a stable version of gatsby here rather than @next

Reply:
Good call, updated this